napari/_vispy/vispy_vectors_layer.py

```diff
@@ -4,9 +4,9 @@
 
 
 class VispyVectorsLayer(VispyBaseLayer):
-    def __init__(self, layer):
+    def __init__(self, layer, MAX_TEXTURE_SIZE=None):
         node = MeshNode()
-        super().__init__(layer, node)
+        super().__init__(layer, node, MAX_TEXTURE_SIZE)
```

Comment:
I'm not a fan of being able to set the value here (and elsewhere)! Is this necessary? Isn't it a limitation of the graphics card?

Reply:
yeah agreed - I don't like this either, will try and fix

Reply:
fixed